src/ComputationContainer/FixedPoint/FixedPoint.hpp

```diff
@@ -103,7 +103,30 @@ class FixedPointImpl : private boost::operators<FixedPointImpl<>>
             ret = std::numeric_limits<D>::max();
         }
         ret /= shift;
-        return ret.str(20, std::ios_base::fixed);
+        return ret.str(0, std::ios_base::fixed);
```

Comment:
Is it needed to out all 50 decimal places?  
I sent patch on https://github.com/acompany-develop/QuickMPC/pull/24#issuecomment-1235035632, but 
I've been wondering if it's really necessary because `FixedPoint` has 8 decimal places. :bow:

If it is needed, bytes sent will be bigger.  
For the reason I'd like to choose other serialization like boost.serialization, cereal etc. as future tasks.

Reply:
Thanks for the comment.
The test was confirmed to pass, so the values were restored!

Reply:
@mdonaka Thank you!